### ISSUE TITLE
Is it possible to expose RDP/VNC ports?

### ISSUE THREAD

**Issue by typkrft** [linuxserver/docker-firefox#8]
---
name: Add VNC/RDP Port
about: It would be nice to access this container directly with RDP or VNC to more easily define user access.

---
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

Expose RDP or VNC ports
------------------------------

## Desired Behavior

Ability to connect to this container from guacamole
## Current Behavior

Currently a light instance of guacamole is used. 
## Alternatives Considered

Similar to the jlesage/firefox which exposes a webui and a VNC port. Their docker has issues with copy and paste.

**Comment by typkrft** [comment]
Lol, I looked through the dockerfile and only saw expose 3000, but then read that it's just metadata. 3389 was there all along. Sorry rookie mistake.

**Comment by destrugter** [comment]
@typkrft Can you help me with how you resolved this? I tried exposing port 3389 but still can't connect via VNC viewer

**Comment by typkrft** [comment]
@destrugter There really shouldn't be much to do. 

According, to the dockerfile documentation on [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) any port that is open can be published. I don't exactly remember why I thought it wasn't working when I opened this. But I think I was looking at the dockerfile before actually using it. I don't use this image anymore though now that  jlesage/firefox  has been updated. 

I also don't recall if this every had a RDP sever running on it, which is what 3389 would be for. For a VNC Viewer you would likely be looking for 5900 more than likely. This image is based on KasmVNC so you might look at that for additional insight. 

This is an example of my compose for jlesage's image that exposes the vnc port so that I can use it through guacamole. 

```dockercompose
  firefox:
    image: jlesage/firefox:v1.17.1
    container_name: firefox
    environment:
      TZ: America/New_York
      KEEP_APP_RUNNING: 1
    ports:
      - 7002:5900
    volumes:
      - /docker_storage/firefox/eztq:/config
    shm_size: "2gb"
    restart: unless-stopped
```

**Comment by typkrft** [comment]
@destrugter There really shouldn't be much to do. 

According, to the dockerfile documentation on [EXPOSE](https://docs.docker.com/engine/reference/builder/#expose) any port that is open can be published. I don't exactly remember why I thought it wasn't working when I opened this. But I think I was looking at the dockerfile before actually using it. I don't use this image anymore though now that  jlesage/firefox  has been updated. 

I also don't recall if this every had a RDP sever running on it, which is what 3389 would be for. For a VNC Viewer you would likely be looking for 5900 more than likely. This image is based on KasmVNC so you might look at that for additional insight. 

This is an example of my compose for jlesage's image that exposes the vnc port so that I can use it through guacamole. 

```dockercompose
  firefox:
    image: jlesage/firefox:v1.17.1
    container_name: firefox
    environment:
      TZ: America/New_York
      KEEP_APP_RUNNING: 1
    ports:
      - 7002:5900
    volumes:
      - /docker_storage/firefox/config:/config
    shm_size: "2gb"
    restart: unless-stopped
```
